ompi/mpi/java/java/Request.java

```diff
@@ -113,6 +117,22 @@ public final void cancel() throws MPIException
 	private native void cancel(long request) throws MPIException;
 
 	/**
+	 * Adds a buffer to the buffer array list.  This method should
+	 * be called by the internal api whenever a persistent request
+	 * is created and any time a request object, that has an associated
+	 * buffer, is returned from a non-blocking operation to protect 
+	 * the buffers from being garbage collected prematurely.
+	 * @param buf buffer to add to the array list
+	 */
+	protected final void addBufRef(Buffer buf)
+	{
+		if(this.buffers == null)
```

Comment:
Minor nits on the style here:

* Did we really use 8 space tabs throughout the Java code?  4 spaces would be preferable, since that's what the rest of the OMPI code base uses.
* `if (null == this.buffers)`: put a space after `if`, and put the constant on the left side of the `==` (i.e., defensive programming)
* Put `{}` around all blocks, including 1-line blocks (i.e., even more defensive programming)

See https://github.com/open-mpi/ompi/wiki/CodingStyle.